### PR TITLE
rough draft summer update

### PR DESCRIPTION
**Pull request by azyth**
@kyleoliveira take a look, not ready for merge yet, but I added a bunch of content, still need to spell check and get some more images for the hardware stuff i did, maybe a screen shot or two from the app, 

You can see the direction I am goign though, any content I should aim to include that i may miss?

Need to record my demo day pitch somehow also.

### CODE REVIEW

File: .idea/workspace.xml [hunk 1]
```diff
@@ -2,7 +2,8 @@
 <project version="4">
```
Comment (kyleoliveira) [inline .idea/workspace.xml]: I think you actually don't want the .idea folder in the repo as it's specifically for project settings in JetBrains products.

1) carefully move the .idea folder somewhere else
2) add a commit that deletes the folder from the repo
3) add a .gitignore (if we don't already have one) and a line with `.idea`
4) move your .idea folder back

After that .idea shouldn't show up when you try to do a `git status`. Note that you should do all that from the terminal.

File: index.html [hunk 2]
```diff
@@ -624,12 +630,14 @@ <h3 class="section-subheading text-muted">Please contact us via the facebook pag
...
-                            <h2>Project Name</h2>
-                            <p class="item-intro text-muted">Lorem ipsum dolor sit amet consectetur.</p>
-                            <img class="img-responsive img-centered" src="img/portfolio/escape-preview.png" alt="">
-                            <p>Escape is a free PSD web template built by <a href="https://www.behance.net/MathavanJaya">Mathavan Jaya</a>. Escape is a one page web template that was designed with agencies in mind. This template is ideal for those looking for a simple one page solution to describe your business and offer your services.</p>
-                            <p>You can download the PSD template in this portfolio sample item at <a href="http://freebiesxpress.com/gallery/escape-one-page-psd-web-template/">FreebiesXpress.com</a>.</p>
-                            <button type="button" class="btn btn-primary" data-dismiss="modal"><i class="fa fa-times"></i> Close Project</button>
+                            <h2>PUMP</h2>
+                            <p class="item-intro text-muted">we having been using Open scad to further develop a design we got of the
```
Comment (kyleoliveira) [inline index.html]: Pretty sure it's `OpenSCAD`...

File: index.html [hunk 3]
```diff
@@ -624,12 +630,14 @@ <h3 class="section-subheading text-muted">Please contact us via the facebook pag
...
+                                internet to produce a pump head that when hooked into a motor housing also under development is capable
+                                of funtioning in any small scale / Home brewery, and replacing existing unsatisfactory pumps in the processes. </p>
```
Comment (kyleoliveira) [inline index.html]: ...is capable of **functioning** in any **small-scale** or **home** brewery....

**Comment by kyleoliveira** [comment]
Also, I'm not sure you need all the extra images. For example, what's with the WordPress logo? We're not using WordPress. Basically, if we don't use it on the page and we're not required to due to some legal thing with the theme, I'd rather leave it out.

**Comment by azyth** [comment]
it was midnight when i uploaded all this stuff, I will be cleaning up the files and stuff when i get everythign set, images file folders, etc.

**Pull request merged**